i18n/translations/en.json

```
@@ -641,6 +641,7 @@
 	"system.issues.https": "We recommend HTTPS for all your sites",
 	"system.issues.kirby": "The kirby folder seems to be exposed",
 	"system.issues.site": "The site folder seems to be exposed",
+	"system.issues.vue.compiler": "The Vue template compiler is enabled",
```

`Suggestion`: To make the warning more actionable:

```suggestion
	"system.issues.vue.compiler": "The Vue template compiler is enabled. Please follow this link and review if it can be disabled for your site to improve security.",
```

(Maybe too long, feel free to change it.)

We could also think about setting the `back` and/or `icon` properties to make it more a warning than an urgent issue.

---

It's a bit long and isn't the second part true for any of the security warnings we list?

Implemented a different theme 👍